daemons/compute/client-go/main.go

```diff
@@ -153,8 +155,12 @@ func createRequest(ctx context.Context, client pb.DataMoverClient, workflow, nam
 	return rsp, nil
 }
 
-func getStatus(ctx context.Context, client pb.DataMoverClient, uid string, maxWaitTime int64) (*pb.DataMovementStatusResponse, error) {
+func getStatus(ctx context.Context, client pb.DataMoverClient, workflow string, namespace string, uid string, maxWaitTime int64) (*pb.DataMovementStatusResponse, error) {
 	rsp, err := client.Status(ctx, &pb.DataMovementStatusRequest{
+		Workflow: &pb.DataMovementCreateRequest_Workflow{
```

Comment:
Regarding my earlier question of a `DataMovementWorkflow` message - it seems a bit weird to create a `CreateRequest_Workflow` to get the status.

Reply:
agreed